src/Domain/QueryBuilders/QueryBuilderInterface.php

```diff
@@ -8,7 +8,9 @@
 
 interface QueryBuilderInterface
 {
-    public function add(string $type, SyntaxInterface $syntax): void;
+    public function must(SyntaxInterface $syntax): void;
 
-    public function build(): array;
```

Comment:
Why is build gone?

Reply:
Because it was already on the SyntaxInterface. If you deduce why it is used and where, you never need to use the `build` command where you actually build the query. Only when you use it for communicating to ElasticSearch, which is what the SyntaxInterface is also for.